opentok-angular.js

```diff
@@ -255,6 +255,7 @@ ng.module('opentok', [])
             props = scope.props() || {};
           props.width = props.width ? props.width : ng.element(element).width();
           props.height = props.height ? props.height : ng.element(element).height();
+          props.preferredResolution = 'auto';
```

Comment:
Why don't we pass this in at the meet layer, instead?

Reply:
We synced with Alberto, these changes need to be in here.